python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gen3authz"
-version = "1.2.0"
+version = "1.3.0"
```

Comment:
technically the "api", or at a function signature, has changed. so calls like `client.auth_request(1, 2, 3, 4)` no longer have the same meaning as before because the order of arguments has changed. I'd consider this to be a major semantic bump since that function is one of the main outputs of this library. Alternatively, you could change the logic below so that existing calls work without updating and then just manually pass `jwt=None` for the update in clients consuming this library

Reply:
Ohh that's a really good point. Im gonna change the logic so that we dont break things.